### DISCUSSION TITLE
pyrightconfig.json does not resolve reportMissingImports error/warning

### DISCUSSION THREAD

**Discussion by dong-bellevue** [microsoft/pyright#7274] [General]
I use visual studio 2022 community version with Anaconda for python work. I have third party packages installed in a virtual environment named BKRCast. In VS IDE, I get this error message:
![image](https://github.com/microsoft/pyright/assets/23224185/68846b5f-e2af-450c-956a-585b26a3eedf)

By following the document, I created pyrightconfig.json with the following two lines:
![image](https://github.com/microsoft/pyright/assets/23224185/6bca24e8-6346-4072-9798-96b559b7657f)

However, it seems this configuration file does not resolve the error. I know the packages are correctly installed and can be imported without any issue when the program is executed in console window or VS IDE. I can ignore this type check message but it would be nice if I can resolve this false alarm. Any comments or suggestions are highly appreciated.

**Comment by erictraut** [comment]
I recommend against using the "venv" and "venvPath" configuration mechanism for specifying your python environment. Please refer to [this documentation](https://microsoft.github.io/pyright/#/import-resolution?id=configuring-your-python-environment) for the preferred ways.

**Reply by dong-bellevue** [reply]
Thank you for your reply. I did try it before I tried `venv` and `venvPath` but it still does not work. 
![image](https://github.com/microsoft/pyright/assets/23224185/09b860fc-9217-45eb-93fc-9a666f7a9b07)

Is this what you meant? I saved this pyrigthconfig.json in the root of my project. I restarted my VS2022 to make sure the changes to take effect. Any suggestions? Thanks!

**Reply by erictraut** [reply]
Could you provide more details? In particular, please provide a minimal, self-contained code sample (in text form). Also, provide the name and version of the package that you're trying to import and (if it's not obvious) installation instructions for that package.
